concourse/artifact/eloqkv.ini

```diff
@@ -1,27 +1,92 @@
+# EloqKV Configuration File
+# For more information, visit: https://www.eloqdata.com/eloqkv/configuration
+
 [local]
-# It is recommended to set the IP to either a private or public IP address
-bind_all=off
+# Whether listen on all interfaces. If bind_all is enabled EloqKV will listen on 0.0.0.0, 
+# otherwise it will listen on the address specified by ip.
+bind_all=false
+
+# Local IP address of the EloqKV node. This value is recorded in the EloqKV catalog and 
+# cannot be changed once the server has been launched. It is strongly recommended to set 
+# this to the actual IP address of the node.
 ip=127.0.0.1
+
+# Listening port for the EloqKV node. Similar to the ip parameter, the port cannot be 
+# changed once the server has been launched.
 port=6379
-# Checkpoint_interval determine the frequency of flushing dirty records into data store
-checkpoint_interval=60
-# Whether enable data store
-enable_data_store=off
-# Whether enable redo log
-enable_wal=off
-# EloqKV data directory. By default, the transaction service data, log service data, and local key-value storage data are all stored in the eloq_data_path directory
-eloq_data_path=eloq_data
```

Comment:
in website, it's name `path`, but i think the `eloq_data_path` is correct, let me update it.